components/Tabs/tests/unit/MDCItemBarTests.m

```diff
@@ -0,0 +1,98 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+#import "MDCItemBar.h"
```

Comment:
Nit/Order of imports: Please place the "MDCItemBar.h" at the top to mark it as the class under test.  I believe that would be the correct application of the [Google Objective-C Style Guide's recommendation to place the "related header" before system framework headers](http://google.github.io/styleguide/objcguide.html#order-of-includes).